recipes/matplotlib-py/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "matplotlib-py" %}
```

Comment:
Matplotlib is already available from conda-forge. Just depend on `matplotlib-base`. You should remove the entire `matplotlib-py` directory.